tests/test_rbd_csi_default_sc.py

```diff
@@ -0,0 +1,208 @@
+"""
+Test for creating a pvc with default RBD StorageClass - CSI
+"""
+import os
+from time import sleep
+
+from kubernetes import client, config
+from ocs import pod
+import yaml
+from ocsci.enums import StatusOfTest
+from utility import templating
+from ocs import ocp
+from ocsci import run_this, EcosystemTest, tier1
+import logging
+
+log = logging.getLogger(__name__)
+
+RBD_POOL_YAML = os.path.join("templates/ocs-deployment", "pool.yaml")
+SC_RBD_YAML = os.path.join("templates/ocs-deployment",
+                           "storageclass-csi-rbd.yaml")
+SECRET_RBD_YAML = os.path.join("templates/ocs-deployment", "secret_rbd.yaml")
+PVC_RBD_YAML = os.path.join("templates/ocs-deployment", "pvc-rbd.yaml")
+TEMP_YAML_FILE = 'test_RbdCSI.yaml'
+
+SC = ocp.OCP(
+    kind='StorageClass', namespace="rook-ceph"
+)
+POOL = ocp.OCP(
+    kind='CephBlockPool', namespace="rook-ceph"
+)
+DEPLOYMENT = ocp.OCP(
+    kind='Deployment', namespace="rook-ceph"
+)
+SECRET = ocp.OCP(
+    kind='Secret', namespace="default"
+)
+PVC = ocp.OCP(
+    kind='PersistentVolumeClaim', namespace="rook-ceph"
+)
+
+def create_rbd_pool():
+    """
+    Create Blockpool with default values
+    """
+    file_y = templating.generate_yaml_from_jinja2_template_with_data(
+        RBD_POOL_YAML
+    )
+    with open(TEMP_YAML_FILE, 'w') as yaml_file:
+        yaml.dump(file_y, yaml_file, default_flow_style=False)
+    log.info(f"Creating a new CephBlockPool with default name")
+    assert POOL.create(yaml_file=TEMP_YAML_FILE)
+    sleep(5)
```

Comment:
please don't use sleep. never (:
Everything that needs waiting, we will sample to check the status